env.tpl

```diff
@@ -48,6 +49,8 @@ REDIS_CACHE_PREFIX=onprem
 REDIS_HOST=redis
 REDIS_PORT=6379
 
+LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
why do we need to add this?

Reply:
In our tests, we stomped with this error:
```bash
ERROR: gcloud failed to load: /tmp/_MEIrcBpgi/libssl.so.1.1: version 'OPENSSL_1_1_1' not found (required by /usr/lib/python3.7/lib-dynload/_ssl.cpython-37m-x86_64-linux-gnu.so)
```

Which we were able to solve with:
```bash
export LD_LIBRARY_PATH=/usr/local/lib
```

We added the variable to the environment to avoid the error